src/open_inwoner/ssd/tests/test_client.py

```diff
@@ -125,6 +126,22 @@ def test_tsl_client_cert_and_private_key(self, mock_request_body, mock_request):
             ),
         )
 
+    def test_requests_exception(self, mock_request_body):
+        self.patched_requests = patch("open_inwoner.ssd.client.requests.post").start()
+        self.patched_requests.side_effect = ConnectionError
+
+        ssd_client = ConcreteSSDClient()
+        ssd_client.config = SSDConfigFactory.build(
+            service__url="https://example.com/soap-service",
+        )
+
+        context = {"bsn": "dummy", "period": "dummy"}
+        response = ssd_client.templated_request(**context)
+
+        self.assertIsNone(response)
+
+        patch.stopall()
```

Comment:
The general pattern for mocking is to use `patch()` as a method decorator or context manager, so the stop/unpatch is always executed.

Example from the project:

    @patch("open_inwoner.pdc.models.mixins.geocode_address", side_effect=IndexError)
    def test_exception_is_handled_when_city_and_postcode_are_not_provided(
        self, mock_geocoding
    ):